android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/DimmedImageBackground.kt

```diff
@@ -0,0 +1,48 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.blur
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color.Companion.Black
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import vn.luongvo.kmm.survey.android.ui.theme.Black20
+
+@Composable
+fun DimmedImageBackground(@DrawableRes imageRes: Int) {
```

Comment:
We might use this with an image URL from API, so we might need to improve this.

Reply:
Yes, but not this time @suho 😛